qa/wildfly/src/test/java/org/elasticsearch/wildfly/WildflyIT.java

```diff
@@ -50,6 +51,7 @@
 import static org.hamcrest.Matchers.equalTo;
 import static org.hamcrest.Matchers.instanceOf;
 
+@TestRuleLimitSysouts.Limit(bytes = 14000)
```

Comment:
we should really add a comment about why this is needed

Reply:
Apologies, I hit "Squash and merge" before :octocat: had refreshed the page and thus missed this comment. I will address in a subsequent PR.